<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AverageColor</class>
 <widget class="QDialog" name="AverageColor">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>495</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>495</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Average Color</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::FormatJustifyCenter"/>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>318</height>
    </rect>
   </property>
   <widget class="QGraphicsView" name="gvImage">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>317</x>
      <y>12</y>
      <width>171</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbLoadImage">
         <property name="text">
          <string>Load Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCalculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSkipTransparent">
       <property name="text">
        <string>Skip transparent pixels</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QGraphicsView" name="gvColor">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
         </property>
         <property name="interactive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leColor">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>#</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
